### PR TITLE
Gen report from check

### PR DESCRIPTION
**Pull request by manelmontilla**
This PR adds a new functionality in the vulcan-security-overview that allows to generate a full report from the output generated by a check.
The functionality is intended to be used while developing a check locally, so it is easy to see how the vulnerabilities generated by the check look in the full report.
This is an example of the command using the new functionality:
```

vulcan-security-overview -config security-overview.toml -check check_report.json 
```

### CODE REVIEW

File: cmd/vulcan-security-overview/main.go [hunk 1]
```diff
@@ -25,10 +26,12 @@ Takes a path to the json file. for instance ./report.json`)
 	assetsURL  = flag.String("assetsurl", "", "[required with regen] specifies the base url where the manage")
 	detailsURL = flag.String("detailsurl", "", "[required with regen] specifies the base url of the details")
 	output     = flag.String("output", "", "[required with regen] specifies the directory to save regenerated report")
+	check      = flag.String("check", "", `generates the security overview for test pourposes from a single check report stored in
```
Comment (filewalkwithme) [inline cmd/vulcan-security-overview/main.go:29]: can you add an example value, like: `ex: -check check_report.json` ? It was not clear to me that this was the json file containing the output of the check execution

File: report.go [hunk 2]
```diff
@@ -140,6 +140,81 @@ func (d *DetailedReport) GenerateLocalFiles() error {
...
+	//    '--<public-bucket>/
+	//	         '
+	//           '--<hex(sha256(teamName))>/
+	//	                '
+	//                  '--<YYYY-MM-DD>/
+	//                         '
+	//                         '--<Most Vulnerable Assets>.png
+	//                         '
+	//                         '--<Impact Distribution>.png
+	d.Email, err = report.GenerateOverview(d.conf, d.awsConfig, d.conf.General.ResourcesPath, d.folder, reportData, d.teamName, d.teamID, d.scanID)
```
Comment (filewalkwithme) [inline report.go:188]: Do we need to generate the `Overview` part here? Maybe we can remove this.

File: vulcan/vulcan.go [hunk 3]
```diff
@@ -114,6 +117,54 @@ func GetReportData(conf config.Config, scanID string) (*ReportData, error) {
...
+// GetReportDataFromFile extracts information about a check report
+// stored in file.
+func GetReportDataFromFile(conf config.Config, scanID, path string) (*ReportData, error) {
+	m := db.NewMemDB()
+	g := groupie.New(m)
+
+	rp := &ReportData{ScanID: scanID, countChecks: 0, groupie: g}
+	date := time.Now().Format("2006-01-02")
+	rp.Date = date
+	log.Printf("Getting reports from results api...")
```
Comment (filewalkwithme) [inline vulcan/vulcan.go:129]: Oh, I think we can replace this log line by `Getting reports from results json file...`

**Review by filewalkwithme — CHANGES_REQUESTED** [review]
Just minor issues. In general the PR looks good.

File: cmd/vulcan-security-overview/main.go (hunk 1 above)
Comment (manelmontilla) [reply cmd/vulcan-security-overview/main.go:29]: Yes thanks for the suggestion!!

File: report.go (hunk 2 above)
Comment (manelmontilla) [reply report.go:188]: Yes true!!

File: vulcan/vulcan.go (hunk 3 above)
Comment (manelmontilla) [reply vulcan/vulcan.go:129]: Yep thanks!!

**Review by filewalkwithme — APPROVED** [review]
LGTM

**Pull request merged**